.github/workflows/merge-develop-into-list-command.yml

```diff
@@ -0,0 +1,13 @@
+name: Merge the develop branch into the list-command branch
+
+on:
+  schedule:
+    - cron: '* 1 * * *'
+jobs:
+  update-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Merge develop -> list-command
+        uses: actions/checkout@v3
+      - run: |
+          curl -X POST -H "Accept: application/vnd.github.v3+json" -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" https://api.github.com/repos/aws/aws-sam-cli/merges -d '{"base":"list-command","head":"develop","commit_message":"Merge develop into list-command"}'
```

Comment:
nit: you can also just use Github CLI to do the same. - https://cli.github.com/manual/gh_api